### PR TITLE
[IMP] web: increase UI Icons font to 14px

### PR DESCRIPTION
**Pull request by Brieuc-brd**
task-3280695

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @stefanorigano 👋  , could you take a look at this ?

File: addons/web/static/src/scss/primary_variables.scss [hunk 1]
```diff
@@ -7,7 +7,8 @@
 $o-webclient-color-scheme: bright !default;
 
 // Font sizes
-$o-root-font-size: 14px !default;
+$o-root-font-base: 14 !default;
```
Comment (stefanorigano) [inline addons/web/static/src/scss/primary_variables.scss]: Where it's used?

Comment (Brieuc-brd) [reply addons/web/static/src/scss/primary_variables.scss]: W'oops ! Forgot to remove this line, fixed !

**Comment by stefanorigano** [comment]
cherry-picked, thanks 👍

**Pull request closed without merging**